src/datumaro/experimental/type_registry.py

```diff
@@ -136,3 +167,103 @@ def from_polars_data(polars_data: Any, target_type: type) -> Any:
     )  # pyright: ignore[reportUnknownMemberType, reportUnknownLambdaType, reportUnknownArgumentType]
 except ImportError:
     pass
+
+
+# Register PIL Image converters if available
+try:
+    from PIL import Image
+
+    register_numpy_converter(Image.Image, lambda x: np.array(x))
+    register_from_polars_converter(
+        Image.Image, lambda x: Image.fromarray(np.array(x).astype(np.uint8))
```

Comment:
Do we have to convert to uint8? Does PIL support other types?

Reply:
I believe PIL only supports uint8 for RGB images. In that case, astype will not do anything since it's already the right type.